config/profiler/requirements.txt

```diff
@@ -0,0 +1,12 @@
+pip=20.3.3
+wheel==0.35.1
+pyYaml==5.3.1
+pytest-html==3.0.0
+sagemaker==2.23.0
+pre-commit==2.6.0
+awscli==1.18.203
+pytest==6.1.2
+pytest-cov==2.10.1
+flaky==3.7.0
+pytest-xdist==2.2.0
+pandas==1.1.5
```

Comment:
Some of them are (sagemaker, flaky, pytest-xdist). We could still move location of files so that eventually all buildspecs can use one requirements.txt? But I think it's best to have separate requirements.txt files if the actual dependencies are different.